service_container/autowiring.rst

```diff
@@ -341,6 +341,15 @@ Thanks to the ``App\Util\TransformerInterface`` alias, the autowiring subsystem
 knows that the ``App\Util\Rot13Transformer`` service should be injected when
 dealing with the ``TransformerInterface``.
 
+.. tip::
+
+    As long as you have only one concrete implementation of an interface under the same 
+    namespace, alias is not mandatory and autowire will work injecting the concrete 
```

Comment:
I would reword this a bit:

> As long as there is only one class implementing the interface and that class is part of the same namespace configuring the alias is not mandatory and Symfony will automatically create one.

Reply:
LGTM, I'm gonna change to suggested sentence.